examples/models/llama2/export_llama_lib.py

```diff
@@ -240,7 +250,8 @@ def quantize(
         from torchao.quantization.quant_api import Int8DynActInt4WeightQuantizer
 
         model = Int8DynActInt4WeightQuantizer(precision=torch_dtype).quantize(model)
-        print("quantized model:", model)
+        if verbose_export():
```

Comment:
If the idea is to guard all prints with this, consider adding a custom print function like `log_export()` that does it internally to simplify the call sites.